soc/espressif/esp32c6/Kconfig

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_ESP32C6
+	select RISCV
+	select RISCV_GP
+	select DYNAMIC_INTERRUPTS
+	select CLOCK_CONTROL
+	select PINCTRL
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_C
+	select RISCV_ISA_EXT_ZICSR
+	select HAS_ESPRESSIF_HAL
+	select XIP if !MCUBOOT
+	select HAS_PM
+	select HAS_POWEROFF
+
+if SOC_SERIES_ESP32C6
+
+config IDF_TARGET_ESP32C6
+	bool "ESP32C6 as target board"
+	default y
+
+config ESP32_PHY_MAX_WIFI_TX_POWER
+	int "Max WiFi TX power (dBm)"
+	range 10 20
+	default 20
+	help
+	  Set maximum transmit power for WiFi radio. Actual transmit power for high
+	  data rates may be lower than this setting.
+
+config ESP32_PHY_MAX_TX_POWER
+	int
+	default ESP32_PHY_MAX_WIFI_TX_POWER
+
+config MAC_BB_PD
+	bool "Power down MAC and baseband of Wi-Fi and Bluetooth when PHY is disabled"
+	depends on SOC_SERIES_ESP32C6 && TICKLESS_KERNEL
+	default n
```

Comment:
```suggestion
```

Reply:
this is not fixed

Reply:
Can you check again? I see this as fixed already